### ISSUE TITLE
Template.dynamic appears to be broken

### ISSUE THREAD

**Issue by dweldon** [mquandalle/meteor-jade#75]
`Ui.dynamic` has been replaced with `Template.dynamic`. However, this:

``` jade
+Template.dynamic(template=template)
```

Gives me this error: `Expected template or null, found: undefined`

``` html
{{> Template.dynamic template=template}}
```

Works correctly if it's in an `.html` file, but not when it's in a `.jade` file.

**Comment by Kestanous** [comment]
Sorry to bump a closed issue but I am having this problem (0.4.1).
```jade
+template.dynamic(template='hello')
```
I tested it in a fresh meteor app with the initial html file converted to jade:

`Uncaught Error: Expected template or null, found: undefined`

The crazy thing is this works:
```jade
+UI.dynamic(template='hello')
```
I am on METEOR@1.1.0.2

**Comment by Taurus-Master** [comment]
I cant get it to work! i changed the value of Template in many ways, but still not working, help please!
```
<body>

    {{> Template.dynamic template=Template data=data}}

</body>

<template name="Dynamic">
    {{#with getTemplate name}}
        {{> template}}
    {{/with}}
</template>

  Template.body.helpers({
    'Template': function() {
      return page.template;
    },
    'data': function() {
      return page.getTemplate();
    }
  });

```

'data' is fake, i was just testing everything.
I'm using the last version,  1.1.0.2
